installer/roles/kubernetes/templates/deployment.yml.j2

```diff
@@ -270,6 +344,15 @@ spec:
               cpu: "{{ task_cpu_limit }}m"
 {% endif %}
         - name: {{ kubernetes_deployment_name }}-redis
+{% if redis_security_context_enabled is defined and redis_security_context_enabled | bool %}
+          securityContext:
+{% if redis_security_context_privileged is defined %}
+            privileged: {{ redis_security_context_privileged }}
+{% endif %}
+{% if redis_security_context_user is defined %}
+            runAsUser: {{ redis_security_context_user }}
```

Comment:
I am not sure about this part. OpenShift (our primary downstream target) will automatically run under a random UID. Can you remove this separate conditional and include it in the `redis_security_context_privileged` conditional above it?

Reply:
Sorry, apparently I cant read jinja. It was hard to tell that this was all wrapped in the outer `if`.